pkg/providers/pricing/zz_generated.pricing_aws.go

```diff
@@ -16,9 +16,11 @@ limitations under the License.
 
 package pricing
 
-// generated at 2024-09-30T13:12:07Z for us-east-1
+import ec2types "github.com/aws/aws-sdk-go-v2/service/ec2/types"
```

Comment:
Does this break any parsing logic that we have? I recall us doing some diff check in our automation that raises PRs when this file changes and I think this import being in this spot might break the automation around this